perllib/FixMyStreet/Cobrand/Westminster.pm

```diff
@@ -116,4 +116,12 @@ sub _fetch_features_url {
     return $cfg->{proxy_url} . "?" . $uri->as_string;
 }
 
+sub categories_restriction {
```

Comment:
Won't that stop fixmystreet.com users in Westminster being able to send reports to TfL?

Reply:
Oh I assumed not wanting them there meant they wanted to get them all instead. Fair enough.
In which case, my comment becomes - I don't see how the XXX happens because categories_restriction happens before bodies_to_list (which is what is shown) is generated, so it feels like there's a bug here somewhere.

Reply:
Ah, it looks like it's because the JS report_form_ajax call uses bodies rather than bodies_to_list - I actually changed that as part of my bugfixing work on Wednesday but it broke a test and I couldn't see if there was a reason it differed that I was missing.